pages/admin/balance.tsx

```diff
@@ -95,7 +95,7 @@ function AdminBalancePage(props) {
   const sidebarElement = <AuthenticatedSidebar active="ADMIN_BALANCE" viewer={props.viewer} />;
 
   return (
-    <Page title="Estuary: Admin: Manage Balance" description="The balance remaining in the Estuary node for Filecoin deals." url="https://estuary.tech/admin/balance">
+    <Page title="Estuary: Admin: Manage Balance" description="The balance remaining in the Estuary node for Filecoin deals." url="/admin/balance">
```

Comment:
You'll want to keep this for SEO, relative URLs don't work for the metatag.

Reply:
Ahhh I wondered what the URIs in the page tags were for. No problem reverting them all, but couldn't we use window.location.hostname in components/Page.tsx so that the url property can be specified as site-relative? Also happy to do that if you think it better.

Reply:
With NextJS you actually can't do that, what you need to do is something like this

```
export async function getServerSideProps(context) {
  return {
    props: { host: context.req.headers.host },
  };
}
```

The rules here are

* If the `.tsx` or `.js` file in the `pages` folder has a module export with the namespace `getServerSideProps` then this code will execute on the server side and not client side. 
* On the server, the way to access the host is as follows: `context.req.headers.host`
* Do not use `window`, its not accessible on the server.

Then you can access host as follows...

```
function YourPageComponent(props) {
  const { host } = props;
  
  return (
     <div>{host}</div>
  )
}

export default YourPageComponent;
```

I appreciate it if you want to do this, this would obviously be an improvement, just a lot of boilerplate obviously :-)

Reply:
Ok, I'll do the getServerSideProps magic. :-)